drivers/hmis_external_apis/app/assets/javascripts/external_forms.js

```diff
@@ -36,6 +47,14 @@ $(function () {
       formData[item.name] = item.value;
     });
 
+    if (window.householdId) {
+      // If form collects household information, add householdId and relationshipToHoH to submission
+      formData['Enrollment.householdId'] = window.householdId;
+      if (!formData['Enrollment.relationshipToHoH']) {
+        formData['Enrollment.relationshipToHoH'] = window.isFamilyMember ? 'DATA_NOT_COLLECTED' : 'SELF_HEAD_OF_HOUSEHOLD';
+      }
+    }
```

Comment:
@martha let's discuss. The form as-written does not ask a question about Relationship To HoH, so I added this. 

However I think it would be safer to also implement this logic in the backend. That way, if someones browser is behaving in a way we don't expect, and householdId is not present for whatever reason, they could still be processed into valid Enrollments (albeit all with exclusively single-member households).

The backend  logic would be something like:
* If HouseholdID present on the submission and is valid (doesnt exist in another project),
  * If relationshipToHoH is also present on the submission, use that
  * Else set relationshipToHoH to 1 if householdid not found, 99 otherwise
* Else,
  * Generate a new HouseholdID, set RelationshipToHoH to 1

Reply:
Thanks @gigxz , makes sense. I've taken a stab at that logic here: https://github.com/greenriver/hmis-warehouse/pull/4657/commits/10c12c6b5724c3788c9b8052263c3e0bdc63ec91

But yes let's discuss...
> If HouseholdID present on the submission

What will the format of this be in the submission values?
```
{
  'Enrollment.householdId': '...',
  ...
}
```
Do we need to add support for a hidden form item in order to submit it as a form value? If not, we probably need to change the behavior of the form processor so it doesn't raise when it encountered an unexpected value based on the form definition. https://github.com/greenriver/hmis-warehouse/blob/bdaf04c01458bb256af19ce3fbfd7b9e397a5f1e/drivers/hmis/app/models/hmis/form/form_processor.rb#L78